apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -0,0 +1,460 @@
+-module(aecore_sync_SUITE).
+
+%% common_test exports
+-export(
+   [
+    all/0, groups/0, suite/0,
+    init_per_suite/1, end_per_suite/1,
+    init_per_group/2, end_per_group/2,
+    init_per_testcase/2, end_per_testcase/2
+   ]).
+
+%% test case exports
+-export(
+   [
+    start_first_node/1,
+    start_second_node/1,
+    start_third_node/1,
+    mine_on_first/1,
+    mine_on_second/1,
+    mine_on_third/1,
+    mine_again_on_first/1,
+    restart_first/1,
+    restart_second/1,
+    restart_third/1
+   ]).
+
+-export([proxy/0]).
+
+-include_lib("common_test/include/ct.hrl").
+
+all() ->
+    [
+     {group, two_nodes},
+     {group, three_nodes}
+    ].
+
+groups() ->
+    [
+     {two_nodes, [],
```

Comment:
The tests in the group appear to require running in order. In order to reduce log content in case of a failing test case, it looks like in a future iteration we may specify [`sequence`](http://erlang.org/doc/apps/common_test/dependencies_chapter.html#sequences) property.